cogs/quotes.py

```diff
@@ -1,48 +1,74 @@
+        "We of the Order of the Straight Arrow call upon the spirit Wematanye, protector of the sacred ground that "
+        "brings us cool water to drink and energy-efficient clean-burning propane gas for all our sacred heating and "
+        "cooking needs. Wematanye says, respect the earth! She's ours, by God, our taxes pay for Her. Also, "
+        "it says here you gotta love all Her creatures. Let's see...oh, here we go: Though we walk through the valley "
+        "of the shadow of death, you're gonna recommend us to the spirit in the sky, with liberty and justice for "
+        "all. Wematanye is with you, and with Texas. Amen.",
         "I am the mack daddy of Heimlich County, I play it straight up, yo. You get the hell outta my 'hood!",
-        "Dang it, I am sick and tired of everyone's asinine ideas about me. I'm not a redneck, and I'm not some Hollywood jerk. I'm something else entirely. I'm... I'm complicated!",
+        "Dang it, I am sick and tired of everyone's asinine ideas about me. I'm not a redneck, and I'm not some "
+        "Hollywood jerk. I'm something else entirely. I'm... I'm complicated!",
         "Firm but with a little give. Yup, these are medium-rare.",
         "You know whats not cool Bobby? Hell.",
-        "Do I Look Like I Know What a JPEG is?"
+        "Do I Look Like I Know What a JPEG is?",
+        "I'm so tired of you South Texas Pig Jockeys comin' in here, wakin' me up to say \"How 'bout them Cowboys!\". "
+        "Arlen stinks and Wichita Falls rules! And you know why? Cause in five minutes I can be in the Great State of "
+        "Oklahoma! Go Sooners!",
+        "This tornado's already registered a level two on the Fujisaki scale. A storm that strong will send an egg "
+        "through a barn door - two barn doors if one of them's open",
+        "I know what's wrong with it. It's a Ford. You know what they say Ford stands for, don't ya? It stands for "
+        "'Fix it again, Tony'",
     ]
-
-    def __init__(self, bot):
-        self.bot = bot
+    SHOWS = {
+        'koth': KOTH
+    }
 
     @commands.group(invoke_without_command=True)
-    async def quote(self, ctx, index: typing.Optional[int] = None):
-        """HuskieBot will say a quote from the best show ever made: King of the Hill"""
+    async def quote(self, ctx, show: typing.Optional[str] = None, index: typing.Optional[int] = None):
+        """HuskieBot will say a quote from show"""
         if ctx.invoked_subcommand is None:
+            if show:
+                try:
+                    show_quotes = self.SHOWS[show]
+                except KeyError:
+                    await ctx.send('That is not a valid show')
+                    return
+            else:
+                show_quotes = self.SHOWS[random.choice(list(self.SHOWS))]
+            if not index:
+                index = randint(0, len(show_quotes) - 1)
```

Comment:
Could we use random.choice for the quote as well? Instead of using randint to generate a random index?

Reply:
Yes